WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -226,7 +226,7 @@
         HistoryListFragment.HistoryItemClickInterface,
         EditPostSettingsCallback,
         PrivateAtCookieProgressDialogOnDismissListener {
-    public static final String ACTION_REBLOG = "reblogAction";		
+    public static final String ACTION_REBLOG = "reblogAction";
```

Comment:
I think there are no real changes to this file so you can revert it :-)

Reply:
yep no real changes, but AS was complaining about trailing tabs so I removed them; if you agree I would keep this, let me know.